clients/src/main/java/org/apache/kafka/common/security/oauthbearer/internals/OAuthBearerSaslClientCallbackHandler.java

```diff
@@ -70,6 +71,8 @@ public void handle(Callback[] callbacks) throws IOException, UnsupportedCallback
         for (Callback callback : callbacks) {
             if (callback instanceof OAuthBearerTokenCallback)
                 handleCallback((OAuthBearerTokenCallback) callback);
+            else if (callback instanceof SaslExtensionsCallback)
+                tryAttachExtensions((SaslExtensionsCallback) callback, Subject.getSubject(AccessController.getContext()));
```

Comment:
Can you name this in the same way as handleCallback(OAuthBearerTokenCallback) -- i.e. handleCallback(SaslExtensionsCallback)?  Unless there is a reason to do it differently?  Maybe you are foreshadowing a move of the method onto the OAuthBearerLoginModule class at some point?  If so, then maybe make the method static, name it handleCallback(), and make the handleCallback(OAuthBearerTokenCallback) symmetric by also accepting a Subject parameter?  I think there is value in making the two methods look very much the same except for the type of callback they accept, so whichever you decide let's make them look the same as much as possible.

Reply:
I wanted to document the code via more explicit names